docs/useCanAccessRecordSources.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: "useCanAccessRecordSources"
+---
+
+# `useCanAccessRecordSources`
+
+This hook calls the authProvider.canAccess() method using react-query for a provided resource and action (and optionally a record).
+
+## Usage
+
+`useCanAccessRecordSources` returns takes an object `{ action, resource, record, sources }` as argument. In addition to react-query result properties, it returns a `canAccess` object that has a property for each provided source determining whether the user has access to it.
```

Comment:
"return takes"? Also, what are the "sources" for?

Reply:
I'll add a prop table instead

Reply:
Actually there's already a prop table